.github/workflows/publish-release.yml

```
@@ -43,14 +41,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
         with:
-          # We check out the release pull request's base branch, which will be
-          # used as the base branch for all git operations.
-          ref: ${{ github.event.pull_request.base.sha }}
+          ref: ${{ github.sha }}
       - uses: actions/cache@v3
         id: restore-build
         with:
           path: ./dist
           key: ${{ github.sha }}
+        # set ignore-scripts to skip prePublish
```

Nit: Maybe worth clarifying that it's the `prepublishOnly` script that we're skipping. Also stating why it's safe to skip

```suggestion
        # Set `ignore-scripts` to skip `prepublishOnly` because the release was built already in the previous job
```